README.md

```diff
@@ -245,6 +245,15 @@ expect([1,2,3]).to.not_contain(4)
 expect({'foo': 'bar'}).to.exclude('baz')
 ```
 
+### throw
+
+Asserts that the target throw an exception
```

Comment:
We might add both, but usually we use it as `to.throw`, so `throw` be more correct :P. But I will add `throws` for the use case `.throws(..)` (without `to`)